Documentation/2.0/admin_guide.md

```diff
@@ -155,3 +155,29 @@ Once you have verified that etcd has started successfully, shut it down and move
 #### Restoring the cluster
 
 Now that the node is running successfully, you can add more nodes to the cluster and restore resiliency. See the [runtime configuration](runtime-configuration.md) guide for more details.
+
+### Client Request Timeout
+
+etcd sets different timeouts for various types of client requests. The timeout value is not tunable now, which will be improved soon(https://github.com/coreos/etcd/issues/2038).
+
+#### Get requests
+
+No timeout is set for get requests, because etcd can always return the result in a short time.
+
+#### Watch requests
+
+Unlimited timeout is set for watch requests. Clients can keep long-polling watch requests as long as they want.
```

Comment:
1. No timeout for watch. Please be consistent with the get case.
2. It is the server decides the timeout, not the client:
"Clients can keep long-polling watch requests as long as they want." -> "etcd will not stop a watch requests until client cancels it. "